Sources/App/Views/PackageController/PackageShow+Model.swift

```diff
@@ -154,6 +158,10 @@ extension PackageShow.Model {
     var gitHubRepositoryUrl: String {
         "https://github.com/\(repositoryOwner)/\(repositoryName)"
     }
+
+    func relativeDocumentationURL(target: String) -> String {
+        return "/\(repositoryOwner)/\(repositoryName)/documentation/\(target.lowercased())"
```

Comment:
Running this locally gives me a 404 when following the documentation url. I think we either need to include the reference fragment (`"/\(repositoryOwner)/\(repositoryName)/\(reference)/documentation/\(target.lowercased())"`) for now or add a route handler/converter/redirect to resolve this to the correct S3 bucket url.

Reply:
Apologies! This is intentional, but I didn't explain it well enough in the PR. The fact they don't work is what this comment was referring to:

> There's more to come to make these links actually work

But this felt like enough for a single PR. Making them work will be in a follow-up PR.